system-linux.c

```diff
@@ -1984,6 +1984,23 @@ system_set_ethtool_pause(struct device *dev, struct device_settings *s)
 	ioctl(sock_ioctl, SIOCETHTOOL, &ifr);
 }
 
+static void
+system_set_ethtool_eee_settings(struct device *dev, struct device_settings *s)
+{
+	struct ethtool_eee eeecmd = {
+		.cmd = ETHTOOL_SEEE,
+		.eee_enabled = s->eee,
+	};
```

Comment:
Just a personal choice and consistency... can you align this to the pattern used up? (memset and setting the values)

Reply:
thanks for the comment, updated as suggested.